Support for try-with-resources -
introduced in Python -
allows us to declare resources to be used in a
try block with the assurance that the resources
will be closed after the execution of that block.